### PR TITLE
Create dashboards and runbooks directories and a runbook example

### PR DESCRIPTION
**Pull request by sradco**
Added README.md files to the dashboards and
runbooks directories and updated the main README.md.

Added the first runbook for VirtAPIDown alert.

Signed-off-by: Shirly Radco <sradco@redhat.com>

### CODE REVIEW

**Comment by sradco** [comment]
@fabiand @simon3z @erkanerol Please review my PR.

**Review by erkanerol — APPROVED** [review]

File: dashboards/VirtAPIDown.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# VirtAPIDown
```
Comment (erkanerol) [inline dashboards/VirtAPIDown.md:1]: @sradco  should not this be under `runbooks` folder?

Comment (sradco) [reply dashboards/VirtAPIDown.md]: @erkanerol Yes. Fixed. Thank you.

**Pull request merged**